experimental/examples/imageviewer/shared/src/commonMain/kotlin/example/imageviewer/Localization.kt

```diff
@@ -0,0 +1,38 @@
+package example.imageviewer
+
+enum class AvailableLanguages {
+    DE,
+    EN;
+}
+
+expect fun getCurrentLanguage(): AvailableLanguages
+
+object EnglishLocalization : Localization {
+    override val appName = "My Memories"
+    override val picture = "Picture:"
+    override val back = "Back"
+    override val takePhoto = "Take a photo 📸"
+    override val addPhoto = "Add a photo"
+    override val kotlinConfName = "Kotlin Conf 2023 🎉"
```

Comment:
The correct name is "KotlinConf" :)

(here and below)

Reply:
Done